packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -159,7 +160,8 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
           stateRef.current.onColumnResizeEnd(item);
           state.setKeyboardNavigationDisabled(false);
         },
-        onChange
+        onChange,
+        disabled: isDisabled
```

Comment:
rather than disabling this, shouldn't we just not render the menu button and not display the resizer on hover?

Reply:
That should work, I just felt like it would be appropriate to give the user the ability to disable a column resizer but we can add that when a use case arises.

Reply:
Nevermind, since I ran into issues [here](https://github.com/adobe/react-spectrum/pull/3422#discussion_r965071551) I'm going to go back to the isDisabled approach